環境変数の調整 および Supabase Keep Alive ワークフローの修正

### 現象

`.github/workflows/supabase-keepalive.yml` が失敗する

原因はおそらく環境変数の埋め込みができていないこと

あと、 下記の環境変数を他でも使用しているものに統一したい（現在は両方に対して同じ値を設定している）
Before
- SUPABASE_URL
- SUPABASE_ANON_KEY

After
- EXPO_PUBLIC_SUPABASE_URL
- EXPO_PUBLIC_SUPABASE_ANON_KEY 

### 再現手順

GHA から手動実行する


### 期待する動作

GHA が正常終了すればよし

### 備考


```
##[debug]Evaluating: secrets.SUPABASE_URL
##[debug]Evaluating Index:
##[debug]..Evaluating secrets:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'SUPABASE_URL'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Evaluating: secrets.SUPABASE_ANON_KEY
##[debug]Evaluating Index:
##[debug]..Evaluating secrets:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'SUPABASE_ANON_KEY'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Evaluating condition for step: 'Lightweight query'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Lightweight query
##[debug]Loading inputs
##[debug]Loading env
Run curl -sf "${SUPABASE_URL}/rest/v1/?limit=0" \
  curl -sf "${SUPABASE_URL}/rest/v1/?limit=0" \
    -H "apikey: ${SUPABASE_ANON_KEY}" \
    -H "Authorization: ***" \
    -o /dev/null
  shell: /usr/bin/bash -e {0}
  env:
    SUPABASE_URL: ***
    SUPABASE_ANON_KEY: ***
##[debug]/usr/bin/bash -e /home/runner/work/_temp/f580f4e1-be4f-4a03-bbb2-4e5547d86b3c.sh
Error: Process completed with exit code 22.

```

--- a/.github/workflows/deploy-testflight.yml
+++ b/.github/workflows/deploy-testflight.yml
@@ -63,6 +63,8 @@ jobs:
         env:
           EXPO_TOKEN: ${{ secrets.EXPO_TOKEN }}
           DEPLOY_MESSAGE: ${{ steps.build.outputs.message }}
+          EXPO_PUBLIC_SUPABASE_URL: ${{ secrets.EXPO_PUBLIC_SUPABASE_URL }}
+          EXPO_PUBLIC_SUPABASE_ANON_KEY: ${{ secrets.EXPO_PUBLIC_SUPABASE_ANON_KEY }}
 
       - name: Discord 通知 (成功)
         if: success()

--- a/.github/workflows/eas-update.yml
+++ b/.github/workflows/eas-update.yml
@@ -26,3 +26,6 @@ jobs:
 
       - name: Create update
         run: eas update --channel preview --message "${{ github.event.head_commit.message }}" --non-interactive
+        env:
+          EXPO_PUBLIC_SUPABASE_URL: ${{ secrets.EXPO_PUBLIC_SUPABASE_URL }}
+          EXPO_PUBLIC_SUPABASE_ANON_KEY: ${{ secrets.EXPO_PUBLIC_SUPABASE_ANON_KEY }}

--- a/.github/workflows/supabase-keepalive.yml
+++ b/.github/workflows/supabase-keepalive.yml
@@ -12,10 +12,13 @@ jobs:
     steps:
       - name: Lightweight query
         run: |
-          curl -sf "${SUPABASE_URL}/rest/v1/?limit=0" \
-            -H "apikey: ${SUPABASE_ANON_KEY}" \
-            -H "Authorization: Bearer ${SUPABASE_ANON_KEY}" \
-            -o /dev/null
+          HTTP_STATUS=$(curl -sS --fail-with-body \
+            -w "%{http_code}" \
+            -o /dev/null \
+            "${EXPO_PUBLIC_SUPABASE_URL}/rest/v1/?limit=0" \
+            -H "apikey: ${EXPO_PUBLIC_SUPABASE_ANON_KEY}" \
+            -H "Authorization: Bearer ${EXPO_PUBLIC_SUPABASE_ANON_KEY}")
+          echo "HTTP status: ${HTTP_STATUS}"
         env:
-          SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
-          SUPABASE_ANON_KEY: ${{ secrets.SUPABASE_ANON_KEY }}
+          EXPO_PUBLIC_SUPABASE_URL: ${{ secrets.EXPO_PUBLIC_SUPABASE_URL }}
+          EXPO_PUBLIC_SUPABASE_ANON_KEY: ${{ secrets.EXPO_PUBLIC_SUPABASE_ANON_KEY }}

